man/man8/zpool.8

```diff
@@ -597,6 +597,16 @@ This space occurs when a LUN is dynamically expanded.
 The amount of fragmentation in the pool.
 .It Sy free
 The amount of free space available in the pool.
+Not all
+.Sy free
+space in the pool will be
+.Sy available
```

Comment:
I think that we could say more specifically that we're talking about the zfs filesystem property `available`.  Also I think we should say what the point of `free`/`allocated` is, and point users to the zfs properties for the cases where that is more appropriate (which is the vast majority).

Maybe something like:
The zpool `free`, `allocated`, and `fragmentation` properties describe space allocation within the pool.  When there is less free space, and when it is more fragmented, it is more difficult to locate free space.  This may result in lower write performance compared to pools with more, less fragmented free space.

In contrast, the zfs.8 `available` property describes how much new data can be written to ZFS filesystems.  The zpool `free` property is not generally useful for this purpose, and can be substantially more than the zfs `available` space.  The discrepancy is due to several factors, including RAIDZ parity; the zfs reservation, quota, refreservation, and refquota properties; and space set aside by spa_slop_shift (see zfs-module-parameters.5).

Reply:
@ahrens : unfortunately, this block is in the `free` description, there's a separate block for each of `allocated` and `fragmentation`.

I'm going to split this. Some of your text into `fragmentation`, some into `free`, some into `allocated`, and cross references between them. -- I think that might work. I'm open to other ideas, but triplicating this text will not work well.